README.md

```diff
@@ -41,96 +73,185 @@ pod "KlaviyoSwift"
+- `value`: A numeric value (`Double`) to associate with this event. For example, the dollar amount of a purchase.
 
-// or setting individual properties
-KlaviyoSDK().set(profileAttribute: .firstName, value: "Blob")
-KlaviyoSDK().set(profileAttribute: .lastName, value: "Jr")
-```
+## Push Notifications
+
+### Prerequisites
 
-Note that the only argument `set(profile:)` takes is a dictionary representing a customer's attributes. This is different from `trackEvent`, which can take multiple arguments.
+* An apple developer [account](https://developer.apple.com/).
+* Configure [iOS push notifications](https://help.klaviyo.com/hc/en-us/articles/360023213971) in Klaviyo account settings.
 
-## Anonymous Tracking Notice
+### Setup
 
-By default, Klaviyo will begin tracking unidentified users in your app once the SDK is initialized. This means you will be able to track events from users in your app without any user information provided. When an email or other primary identifier is provided, Klaviyo will merge the data from the anonymous user to a new identified user.
+* Enable push notification capabilities in your Xcode project. The section "Enable the push notification capability" in this [Apple developer guide](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns#2980170) provides detailed instructions.
+* If you intend to use rich push notifications add a [Notification service extension](https://developer.apple.com/documentation/usernotifications/unnotificationserviceextension) to your xcode project.
+A notification service app extension ships as a separate bundle inside your iOS app. To add this extension to your app:
+  * Select File > New > Target in Xcode.
+  * Select the Notification Service Extension target from the iOS > Application extension section.
+  * Click Next.
+  * Specify a name and other configuration details for your app extension.
+  * Click Finish.
+
+> ⚠️ By default the deployment target of your notification service extension might be the latest iOS version and not
+the minimum you want to support. This may cause push notifications to not show the attached media in devices whose
+iOS versions are lower than the deployment target of the notification service extension. ⚠️
 
-Prior to version 1.7.0, the Klaviyo SDK used the [Apple identifier for vendor (IDFV)](https://developer.apple.com/documentation/uikit/uidevice/1620059-identifierforvendor) to facilitate anonymous tracking. Starting with version 1.7.0, the SDK will use a cached UUID that is generated when the SDK is initialized. For existing anonymous profiles using IDFV, the SDK will continue to use IDFV, instead of generating a new UUID.
+### Collecting Push Token
 
-## Profile properties and Identifiers
+In order to send push notifications to your users, you must collect their push tokens and register them with Klaviyo.
+This is done via the `KlaviyoSDK().set(pushToken:)` method, which registers push token and current authorization state
```

Comment:
Should be "which registers a push token"

Reply:
Done.